inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/FetcherConstants.java

```diff
@@ -53,12 +53,13 @@ public class FetcherConstants {
     public static final String AGENT_MANAGER_REPORTSNAPSHOT_HTTP_PATH = "agent.manager.reportsnapshot.http.path";
     public static final String DEFAULT_AGENT_MANAGER_REPORTSNAPSHOT_HTTP_PATH = "/agent/reportSnapshot";
 
+    public static final String AGENT_MANAGER_HEARTBEAT_HTTP_PATH = "agent.manager.heartbeat.http.path";
+    public static final String DEFAULT_AGENT_MANAGER_HEARTBEAT_HTTP_PATH = "/openapi/heartbeat/report";
```

Comment:
The `AGENT_MANAGER_VIP_HTTP_PREFIX_PATH` already contains `/openapi`, so there may not need `/openapi` again.

Reply:
done